### DISCUSSION TITLE
Why livekit protocol as a git submodule

### DISCUSSION THREAD

**Discussion by exalted** [zed-industries/zed#6697] [Feature Requests]
I am curious, why livekit protocol was added as a git submodule rather than adding [from crates.io](https://crates.io/crates/livekit-protocol).

**Comment by maxdeviant** [comment]
We're using the LiveKit Swift SDK and not the Rust one at the moment.

The LiveKit Rust SDK currently requires Tokio, which we don't use in Zed right now:

> Currently, Tokio is required to use this SDK, however we plan to make the async executor runtime agnostic.

**Reply by exalted** [reply]
Ah! Thanks for the explanation, @maxdeviant . Keep up the GREAT work 💪
